client/api_payment.go

```diff
@@ -43,6 +51,18 @@ func (c *Client) GetStreamRecord(ctx context.Context, streamAddress string) (*pa
 }
 
 // Deposit deposits BNB to a stream account.
```

Comment:
Can we change stream account to payment account, only expose one concept to users?

Reply:
Fixed